pkg/controllers/nodeclass/status/instance_profile/controller.go

```diff
@@ -12,62 +12,44 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package status
+package instance_profile
```

Comment:
Package names shouldn't include underscores, same comment for `launch_template` and `security_group`.